src/redux/constants/actionTypes.js

```diff
@@ -18,3 +18,9 @@
 export const FOCUS_TRENDS_CBG_SLICE = 'FOCUS_TRENDS_CBG_SLICE';
 export const MARK_TRENDS_VIEWED = 'MARK_TRENDS_VIEWED';
 export const UNFOCUS_TRENDS_CBG_SLICE = 'UNFOCUS_TRENDS_CBG_SLICE';
+
+// from blip's redux implementation
+// TODO: how could we DRY this out??
```

Comment:
Yeah, I can't think of another way to handle "gracefully" other than a common package. I fear it may come up more with the viz-blip interface, but probably unlikely that it'll happen with more than a half dozen action types, so perhaps that'll all right, not a huge quantity of redundant code.